src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -3,24 +3,41 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+    xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+    
     Title="SonarLint Connected Mode Migration"
     HorizontalAlignment="Center"
     ResizeMode="NoResize"
     SizeToContent="WidthAndHeight"
     MinHeight="400"
-    MaxHeight="600"
     MinWidth="400"
-    MaxWidth="600">
+    MaxWidth="600"
+    
+    Background="{DynamicResource {x:Static vsShell:VsBrushes.ComboBoxBackgroundKey}}"
```

Comment:
The Issue Viz tool window uses `BrandedUIBackgroundKey` for the background i.e. the same colour as the editor. That makes it difficult to see the edges of the dialogue window (black on black in dark theme).

The combobox background is a little bit lighter so the dialogue window stands out more.